### PR TITLE
feat: add cli scripts to quickly run a calculation

### PR DESCRIPTION
**Pull request by yanghan234**
Add
- [x] script to do structure relaxation
- [x] script to compute phonon dispersion
- [x] scirpt to make single point predictions of given structures

### CODE REVIEW

File: cli/applications/phonon.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+            atoms=atoms,
+            find_prim=find_prim,
+            work_dir=work_dir,
+            amplitude=amplitude,
+            supercell_matrix=supercell_matrix,
+            qpoints_mesh=qpoints_mesh,
+            max_atoms=max_atoms,
+        )
+        phonon.run()
+        pred_phonon_list.append(phonon)
```
Comment (bi-ran) [inline cli/applications/phonon.py:59]: should this return the output of `phonon.run()`?

Comment (yanghan234) [reply cli/applications/phonon.py:61]: You are right. I have fixed it.

File: src/mattersim/cli/applications/moldyn.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+
+def moldyn(
+    atoms_list: List[Atoms],
+    *,
+    temperature: float = 300,
+    timestep: float = 1,
+    steps: int = 1000,
+    ensemble: str = "nvt_nose_hoover",
+    logfile: str = "-",
```
Comment (bi-ran) [inline src/mattersim/cli/applications/moldyn.py:23]: could you change this to match the default in `src/mattersim/cli/mattersim_app.py`: `logfile: str = md.log`

File: src/mattersim/cli/applications/moldyn.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    loginterval: int = 10,
+    trajectory: str = None,
```
Comment (bi-ran) [inline src/mattersim/cli/applications/moldyn.py:25]: same here: `trajectory: str = md.traj`

**Review by bi-ran — APPROVED** [review]

**Pull request merged**